docs/src/components/ImgCard/index.js

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import Link from '@docusaurus/Link';
+import styles from "./index.module.scss";
 
 function ImgCard({ dataList }) {
   return (
-    <div className="img-wrapper">
+    <div className={styles.wrapper}>
       {dataList.map((data) => {
         return (
-          <Link className="img-block" to={data?.url} key={data?.title}>
-            <img src={data?.Svg} />
+          <Link to={data?.url} key={data?.title} className={styles.link}>
+            <img src={data?.Svg} className={styles.img} />
```

Comment:
We use `classes` instead of `styles` on the website. Can we use the same for consistency?

Reply:
Same in other places as well

Reply:
Website is using `material-ui`  where normal conventions is to use `classes`. Docs code is totally based upon `docusaurus` where we haven't used the material ui instead used mix of `scss` and `scss-module` . Normally `css-module`'s convention is to use `styles` ( as name ).

I am agree that we are using single repo for both the website source code but still both the code are different ( one is based on react, typescript and material ui && second one is based upon docusaurus, react, javascript and scss, scss-in-module )

Also we are not using the concept of mono-repo ( where some components would have been common in both website )

Reply:
Ok